packages/cli/src/UserManagement/routes/passwordReset.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable import/no-cycle */
+
+import express = require('express');
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
why do we need to keep the version in the variable? updating it to `v5` would be super easy if we don't
```suggestion
import { v4 as uuid } from 'uuid';
```

Reply:
@ivov UUID versions are more like variants. You be unlikely to swap one out for another once deciding on requirements. So it may be useful to understand which version is being used throughout the code.

https://datatracker.ietf.org/doc/html/rfc4122#section-4.4